packages/core/ui/action-bar/index.android.ts

```diff
@@ -7,13 +7,18 @@ import { colorProperty } from '../styling/style-properties';
 import { ImageSource } from '../../image-source';
 import * as application from '../../application';
 import { isAccessibilityServiceEnabled, updateContentDescription } from '../../accessibility';
+import type { Background } from 'ui/styling/background';
```

Comment:
```suggestion
import type { Background } from '../styling/background';
```

Believe that should be a relative import